fixtures/app_with_spm_dependencies/App/Project.swift

```diff
@@ -30,19 +30,6 @@ let project = Project(
                 .sdk(name: "c++", type: .library, status: .required),
                 .external(name: "Alamofire"),
                 .external(name: "ComposableArchitecture"),
-                .external(name: "FacebookCore"),
-                .external(name: "FirebaseAnalytics"),
```

Comment:
Should we consider keeping Firebase in given this dependency is a popular one that we broke in the past? Ofc depending on how much CI time it adds.